dali/kernels/signal/fft/stft_gpu.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```
across the files in PR.